domain/lib/src/usecases/share/share_document_view_state.dart

```diff
@@ -35,8 +35,6 @@ import 'package:domain/domain.dart';
 class ShareDocumentViewState extends ViewState {
   final List<Share> sharedRecipients;
 
-  Document get sharedDocument => sharedRecipients.first?.document;
```

Comment:
`Share` class is using also when you share a file. So document is returned in this case. But actually we don't use it anywhere for now. So should we delete it? @hoangdat

Reply:
Good idea. please discuss it in daily meeting